src/main/java/reposense/system/StreamGobbler.java

```diff
@@ -1,15 +1,14 @@
 package reposense.system;
 
-import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStream;
-import java.io.InputStreamReader;
-
 
 public class StreamGobbler extends Thread {
 
+    private static final byte[] BUFFER = new byte[1 << 13];
```

Comment:
rationale for using 2^13?

Reply:
Based on my own testing, around 2^11 - 2^13 gives the best speed.

Reply:
Hmm, can you explore using [ByteBuffer](https://docs.oracle.com/javase/7/docs/api/java/nio/ByteBuffer.html)?